codecov.yml

```diff
@@ -34,11 +34,11 @@ coverage:
         flags:
           - workbench-core-authentication
       workbench-core-authorization:
-        target: 100%
+        target: 97%
         flags:
           - workbench-core-authorization
       workbench-core-base:
-        target: 90%
+        target: 70%
```

Comment:
Currently it's at 99.52% right? Can we update this value to 95%?

Reply:
Hmm, maybe make this 95 and update the jest config too.

Reply:
workbench-core-base is at 81.13%, according to codecov.